wfl-slack-app/src/slack-api.js

```diff
@@ -0,0 +1,24 @@
+var fetch = require('isomorphic-fetch');
+var queryString = require('query-string');
+
+module.exports = {
+    getUserInfo: function(userId) {
+        var params = queryString.stringify({
+            token: process.env.SLACK_APP_TOKEN,
+            user: userId
+        });
+
+        return fetch(`https://slack.com/api/users.info?${ params }`)
```

Comment:
I'm curious what your thoughts are regarding isomophic-fetch?  How did you choose this library for REST API processing?

Reply:
I used isomorphic-fetch in `slack-api.js` because that's how we interacted with slack from the work from home app and I could reuse this bit of code. I could use request here as well.